doc/asciidoc/faq/index.adoc

```diff
@@ -0,0 +1,167 @@
+
+== Neo4j Streams FAQ
+
+How to integrate Neo4j and Kafka::
+
+When integrating Neo4j and Kafka using Neo4j Streams plugin or Kafka Connect plugin
+is important configure just one of them and not both.
+If you need to load data from Kafka to Neo4j (and not viceversa) you can just use the Kafka Connect plugin.
+If you need to have both sink and source functionalities then you have to use the Neo4j Streams plugin.
+This is the main difference between those plugins.
+
+About CUD file format::
+
+The CUD file format is JSON file that represents Graph Entities (Nodes/Relationships) and how to manage them in term
+of **C**reate/**U**pdate/**D**elete operations.
+So every JSON event represents a single operation.
+
+How to ingest events using CDC Schema strategy::
+
+Change Data Capture method allows to ingest events between different Neo4j instances.
+If you decide to use the Neo4j Streams plugin, then the Neo4j source instance will be configured as follow:
++
+[source, ini]
+----
+streams.sink.enabled=false
+streams.source.schema.polling.interval=10000
+----
++
+and the Neo4j sink instance will be configured as follow:
++
+[source, ini]
+----
+streams.source.enabled=false
+streams.sink.topic.cdc.schema=<topic-name>
+streams.sink.enabled=true
+----
++
+If you decide to use Kafka Connect plugin for the sink instance, then it has to be configured as follow:
++
+[source, json]
+----
+{
+  "name": "Neo4jSinkConnector",
+  "config": {
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "org.apache.kafka.connect.json.JsonConverter",
+    "value.converter.schemas.enable": false,
+    "topics": "<list_of_topics_separeted_by_comma>",
+    "connector.class": "streams.kafka.connect.sink.Neo4jSinkConnector",
+    "errors.retry.timeout": "-1",
+    "errors.retry.delay.max.ms": "1000",
+    "errors.tolerance": "all",
+    "errors.log.enable": true,
+    "errors.log.include.messages": true,
+    "neo4j.server.uri": "bolt://<neo4j-sink-hostname>:7687",
+    "neo4j.authentication.basic.username": "neo4j",
+    "neo4j.authentication.basic.password": "password",
+    "neo4j.encryption.enabled": false,
+    "neo4j.topic.cdc.schema": "<list_of_topics_separeted_by_semicolon>"
+  }
+}
+----
+
+Is Neo4j Streams supported by Confluent Cloud?::
```

Comment:
Add just a bit to this question.  The big thing people want to know is if they can run the connector as a managed service, and the answer to that is no.  It works with Confluent Cloud, but they still need some hardware/software they manage to run the Neo4j connector.

Users who are interested in running Neo4j-Streams as a cloud managed connector by Confluent should request this of confluent.  Right now there are only a few connectors such as that for S3 that can be run as managed services.  https://docs.confluent.io/current/cloud/connectors/index.html

Reply:
I'll update this FAQ with your detailed explanation. Thanks!